edb/server/protocol/auth_ext/ui/components.py

```diff
@@ -336,3 +336,269 @@ def success_message(message: str) -> str:
+                  direction: ltr;
+                  font-size: 0px;
+                  padding: 20px 0;
+                  padding-bottom: 0px;
+                  padding-top: 20px;
+                  text-align: center;
+                "
+              >
+                <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:600px;" ><![endif]-->
+                <div
+                  class="mj-column-per-100 mj-outlook-group-fix"
+                  style="
+                    font-size: 0px;
+                    text-align: left;
+                    direction: ltr;
+                    display: inline-block;
+                    vertical-align: top;
+                    width: 100%;
+                  "
+                >
+                  <table
+                    border="0"
+                    cellpadding="0"
+                    cellspacing="0"
+                    role="presentation"
+                    style="vertical-align: top"
+                    width="100%"
+                  >
+                    <tbody>
+                      <tr>
+                        <td
+                          align="center"
+                          style="
+                            font-size: 0px;
+                            padding: 10px 25px;
+                            padding-top: 0;
+                            padding-right: 0px;
+                            padding-bottom: 0px;
+                            padding-left: 0px;
+                            word-break: break-word;
+                          "
+                        >
+                          <table
+                            border="0"
+                            cellpadding="0"
+                            cellspacing="0"
+                            role="presentation"
+                            style="border-collapse: collapse; border-spacing: 0px"
+                          >
+                            <tbody>
+                              <tr>
+                                <td style="width: 150px">
+                                  <img
+                                    alt="
+                                      {f'{app_name} logo' if app_name else ''}
+                                    "
+                                    height="150"
+                                    src="{logo_url}"
+                                    style="
+                                      border: none;
+                                      display: block;
+                                      outline: none;
+                                      text-decoration: none;
+                                      height: 150px;
+                                      width: 100%;
+                                      font-size: 13px;
+                                    "
+                                    width="150"
+                                  />
```

Comment:
I went with 150x150 here for lack of a better idea. Another option is to have the width set to 150 and have the height be `auto` which would preserve the aspect ratio. We don't give much guidance to the developer about aspect ratio and size, and I think we should.